modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,111 @@
+[[maintenance-windows]]
+= Maintenance Windows
+
+The maintenance windows feature in {productname} allows you to schedule actions to occur during a scheduled maintenance window period.
+When you have created your maintenance window schedule, and applied it to a client, you are prevented from executing some actions outside of the specified period.
+
+
+[NOTE]
+====
+Maintenance windows operate in a different way to system locking.
+For more information about system locks, see xref:client-configuration:system-locking.adoc[].
+====
+
+
+Maintenance windows require both a calendar, and a schedule.
+The calendar defines the date and time of your maintenance window events, including recurring events, and must be in [path]``ical`` format.
+The schedule uses the events defined in the calendar to create the maintenance windows.
+You must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar, before you can create the schedule.
+
+When you have created the schedule, you can assign it to clients that are registered to the {productname} Server.
+Clients that have a maintenance schedule assigned cannot run restricted actions outside of maintenance windows.
+
+Restricted actions significantly modify the client, and could potentially cause the client to stop running.
+Some examples of restricted actions are:
+
+* Package installation
+* Client upgrade
+* Service pack migration
+* Highstate application (for Salt clients)
+
+Unrestricted actions are minor actions that are considered safe and are unlikely to cause problems on the client.
+Some examples of unrestricted actions are:
+
+* Package profile update
+* Hardware refresh
+* Subscribing to software channels
```

Comment:
Should we add a complete list of action which are restricted and which not?
AFAIK Pau wanted to have it.

Reply:
So @hustodemon and I thought that because the list is quite long, we would need to add it at the end of the section or something, rather than here. Happy to do that, if @paususe wants it.

Reply:
I can give you the list. @paususe do you want to have the full list?

Reply:
Yes please, full list.

Reply:
OK. I'll need that list then, please @hustodemon :slightly_smiling_face:

Reply:
@Loquacity I made a list of actions which can only be executed when a client is in maintenance mode (in a maintenance window). I tried to make the list understanable by humans and grouped things togther . We will see if it is correct enough.

* package operations (install/update/remove)
* patch update
* reboot
* config management changing tasks
* State Highstate
* autoinstallation (reinstall)
* remote command
* SP Migration
* Cluster operations

Reply:
lol, I thought the list was long? :joy: